ephios/core/signup/methods.py

```diff
@@ -275,6 +275,8 @@ def check_conflicting_shifts(method, participant):
 
 
 class BaseSignupMethod:
+    # pylint: disable=too-many-public-methods
```

Comment:
We could just reduce the public interface. No outside user needs to see the list of signup_checkers, only subclasses. Hence make the corresponding methods protected (prepend an underscore to the identifier)...